Dockerfile

```diff
@@ -1,20 +1,33 @@
-FROM phusion/baseimage:latest
+FROM edxops/xenial-common:latest
+MAINTAINER stv <stv@stanford.edu>
 RUN apt-get update && apt-get install -y \
-    build-essential \
-    git \
+    gettext \
     lib32z1-dev \
     libjpeg62-dev \
-    libxml2-dev \
     libxslt-dev \
-    python-dev \
```

Comment:
Everything removed from this list is confirmed to already be provided by the new base box :)